app/src/main/java/at/roteskreuz/stopcorona/model/repositories/QuarantineRepository.kt

```diff
@@ -464,25 +465,25 @@ class QuarantineRepositoryImpl(
             .map { (dateOfFirstMedicalConfirmation, dateOfLastSelfDiagnose, areMissingKeysUploaded) ->
                 if (dateOfFirstMedicalConfirmation.isPresent &&
                     ZonedDateTime.now()
-                        .isAfter(dateOfFirstMedicalConfirmation.get().plusDays(1).startOfTheDay()) &&
+                        .isAfter(dateOfFirstMedicalConfirmation.get().endOfTheUtcDay()) &&
```

Comment:
Do we need to convert it to UTC? The reporting date is stored in the local time

Reply:
we do not convert. we need the beginning of the day in UTC-Zone. That's when the keys are rotated